webpack/components/PreupgradeReportsList/components/helpers.js

```diff
@@ -17,12 +43,28 @@ export const additionalInfo = entry => {
     }
 
     switch (attr) {
-      case 'title':
-        return <StringInfoItem entry={entry} attr={attr} key={key} />;
       case 'severity':
+        return  <ListView.InfoItem key={key}>
+                    <Icon
+                        key={key}
+                        type="pf"
+                        name="resources-full"
+                        style={{ color: presentSeverity(entry.severity) }}> {entry.severity}</Icon>
+                </ListView.InfoItem>  
+      case 'hostname':
         return <StringInfoItem entry={entry} attr={attr} key={key} />;
       default:
         return '';
     }
   });
 };
+
+export const tagInfo = entry => {
+  
+  let data = JSON.parse(entry);
```

Comment:
I am getting: 

![tags](https://user-images.githubusercontent.com/2664090/78530602-414c7500-77e4-11ea-9219-c65a976ed12a.png)

Tags no longer have to be parsed as they are serialized properly: https://github.com/oamg/foreman_leapp/commit/0d734aee578c47b527e14614fc8f0679e26cb331